ui/react-components/helper.js

```diff
@@ -33,3 +33,23 @@ export const isServiceTypeEnabled = appConfig => {
 export const getValidProviders = providers => {
     return providers && providers.filter(provider => provider.response === PROVIDER_RESPONSES.ACCEPTED);
 };
+
+export const searchFeildOnChangeHandler=(state,setState,selectedState,setSelectedState,e)=>{
+    setSelectedState([
+        ...selectedState,
+        e
+      ]);
+      setState(() =>
+        [...state].filter(item => item != e)
```

Comment:
What does `e` refers to here? Is it `element` or `event`?
IMO while this function solves the duplication problem, it is a bit tight on the readability part. Let me know your thoughts.
CC: @binduak @bsneha90

Reply:
It's a value changed on event. Modified with more specific name eventChangedValue